pkg/util/volume_util_windows.go

```diff
@@ -0,0 +1,89 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+)
+
+var _ VolumeUtil = &volumeUtil{}
+
+type volumeUtil struct {
+	// csiProxy is the CSIProxy implementation (either v1 or v1beta)
+	csiProxy CSIProxy
+}
+
+// NewVolumeUtil returns a VolumeUtil object for performing local filesystem operations
+func NewVolumeUtil() (VolumeUtil, error) {
+	csiProxy, err := NewCSIProxy()
+	if err != nil {
+		return nil, err
+	}
+	return &volumeUtil{
+		csiProxy,
+	}, nil
+}
+
+// GetFsCapacityByte returns capacity in bytes about a mounted filesystem.
+// mountPath is the pathname of any file within the mounted filesystem. Capacity
+// returned here is total capacity.
+func (u *volumeUtil) GetFsCapacityByte(mountPath string) (int64, error) {
+	volumeID, err := u.csiProxy.GetVolumeId(mountPath)
+	if err != nil {
+		return 0, err
+	}
+	totalBytes, _, err := u.csiProxy.GetVolumeStats(volumeID)
+	if err != nil {
+		return 0, err
+	}
+	return totalBytes, nil
+}
+
+// DeleteContents deletes all the contents under the given directory
+func (u *volumeUtil) DeleteContents(mountPath string) error {
+	volumeID, err := u.csiProxy.GetVolumeId(mountPath)
+	if err != nil {
+		return err
+	}
+	err = u.csiProxy.FormatVolume(volumeID)
```

Comment:
it will format with ntfs, right?
what if previous format type is not ntfs?

Reply:
> it will format with ntfs, right?
yes, that's the default format of the CSI Proxy VolumeFormat call

> what if previous format type is not ntfs?
I formatted a VHD with fat32 and then simulated the `deleteContents` call by doing a reformat with ntfs, it didn't complain:

```
PS C:\tmp> Get-VHD -Path $vhdPath | Get-Partition | Get-Volume | Format-Volume -Filesystem fat32 -Confirm:$false

DriveLetter FriendlyName FileSystemType DriveType HealthStatus OperationalStatus SizeRemaining       Size
----------- ------------ -------------- --------- ------------ ----------------- -------------       ----
                         FAT32          Fixed     Healthy      OK                   1003.93 MB 1003.94 MB


PS C:\tmp> Get-VHD -Path $vhdPath | Get-Partition | Get-Volume | Format-Volume -Filesystem ntfs -Confirm:$false

DriveLetter FriendlyName FileSystemType DriveType HealthStatus OperationalStatus SizeRemaining       Size
----------- ------------ -------------- --------- ------------ ----------------- -------------       ----
                         NTFS           Fixed     Healthy      OK                    991.11 MB 1007.93 MB

```

Reply:
so after user delete pvc, the disk becomes empty disk with ntfs no matter what the previous fs. Maybe it is ok, but need release note about it, I think..

Reply:
release note updated